### ISSUE TITLE
[Feature Request]: Add TF example to python notebook

### ISSUE THREAD

**Issue by ryanthompson591** [apache/beam#23091]
### What would you like to happen?

We have example code for how to use runinference with sklearn and pytorch. See:

https://github.com/apache/beam/tree/master/examples/notebooks/beam-ml

Since we support TF through TFX -BSL it would make sense to add a TF example to that notebook.

I also had a discussion with @AnandInguva around perhaps breaking these notebooks up into three separate ones in the future. It's likely developers will only be interested in a single framework so it might make sense to structure them with that in mind.

### Issue Priority

Priority: 2

### Issue Component

Component: sdk-py-core

**Comment by ryanthompson591** [comment]
.take-issue

**Comment by AnandInguva** [comment]
cc: @rezarokni

**Comment by ryanthompson591** [comment]
.add-labels run-inference

**Comment by AnandInguva** [comment]
I feel like it would be better to have individual notebook for each framework. This looks too big.

**Comment by damccorm** [comment]
> I feel like it would be better to have individual notebook for each framework. This looks too big if we add to a single notebook. Any thoughts?

I agree - I imagine most people new to the transform (or who want to use it for a new purpose) are going to have a specific framework in mind. I would vote we split this out into its own notebook and then follow up to split out sklearn as well.

**Comment by damccorm** [comment]
One recommendation (either as part of this PR or a separate one): it would be nice to have a very simple top level README describing (1) RunInference very generally/pointing to the docs and (2) what each notebook does (including the spaCy one) so that people who are just getting started know where to go. If if its not convenient for that to make it into this PR, I can also follow up on that separately.

**Comment by rezarokni** [comment]
+1 to creating multiple notebooks 
+1 to top level README

**Comment by damccorm** [comment]
@ryanthompson591 can we close this one?
